sbc_compassion/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+"id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
+"access_sponsorship_correspondence","Full access on sponsorship_correspondence","model_sponsorship_correspondence","child_compassion.group_sponsorship",1,1,1,1
+"access_res_lang_compassion","Full access on res_lang_compassion","model_res_lang_compassion","child_compassion.group_sponsorship",1,1,1,1
```

Comment:
Make the group_sponsorship have only read access and add a rule for group admin to have full access.